administration/src/bp-modules/user-settings/ApiTokenSettings.tsx

```diff
@@ -0,0 +1,142 @@
+import { Button, H2, H4, HTMLSelect, HTMLTable } from '@blueprintjs/core'
+import Delete from '@mui/icons-material/Delete'
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+
+import getMessageFromApolloError from '../../errors/getMessageFromApolloError'
+import {
+  ApiTokenMetaData,
+  useCreateApiTokenMutation,
+  useDeleteApiTokenMutation,
+  useGetApiTokenMetaDataQuery,
+} from '../../generated/graphql'
+import { formatDate } from '../../util/formatDate'
+import { useAppToaster } from '../AppToaster'
+import getQueryResult from '../util/getQueryResult'
+import SettingsCard from './SettingsCard'
+
+const Container = styled.div`
+  background: ghostwhite;
+  padding: 20px;
+  border-radius: 8px;
+  box-shadow: inset 0 2px 4px rgba(0, 0, 0, 0.05);
+`
+
+const Row = styled.div`
+  width: 80%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 15px;
+`
+
+const NewTokenText = styled.p`
+  font-size: 18px;
+  color: #007bff;
+  background: #e9f7ff;
```

Comment:
please check my design suggestion or use sth with better contrast between text and background